File: Python/MoveZerosToEnd/MySolution.py
def move_zeros(lst):
    return [x for x in lst if x != 0] + [0] * lst.count(0)

# Original kata: https://www.codewars.com/kata/52597aa56021e91c93000cb0
# My solution: https://www.codewars.com/kata/reviews/57cb52c733c32f4c48000076/groups/62c3d3934b4757000175792b

File: PHP/PersistentBurger/MySolution.php
<?php

function persistence(int $num): int {
    if ($num >= 10) {
        $count = 0;
        while ($num >= 10) {
            $product = 1;
            while ($num > 0) {
                $product *= $num % 10;
                $num = (int)($num / 10);
            }
            $num = $product;
            $count++;
        }
        return $count;
    }
    return 0;
}

// Original kata: https://www.codewars.com/kata/55bf01e5a717a0d57e0000ec
// My solution: https://www.codewars.com/kata/reviews/582bdd4742fdd8d4be000010/groups/68355b28a6f89e7b6b23e8c4